optaplanner-docs/src/main/asciidoc/QuickStart/CloudBalancingBeyondThisTutorial/CloudBalancingBeyondThisTutorial-section.adoc

```diff
@@ -2,8 +2,8 @@
 = Beyond this Tutorial
 :imagesdir: ../..
 
-Now that this simple example works, try going further.
-Enrich the domain model and add extra constraints such as these:
+Now that this simple example works, you can try going further.
```

Comment:
Does product-docs guidelines stipulate to favore active tense ("try", "walk") over using "you" ("you can try", "you can walk")?

Reply:
In  a word, yes. Being directive is encouraged.